### PR TITLE
Chat and comments

### PR DESCRIPTION
**Pull request by wimoMisterX**

### CODE REVIEW

File: root/package.json [hunk 1]
```diff
@@ -6,18 +6,21 @@
...
+    "cookie-parser": "^1.4.0",
     "express": "3.2.3",
     "jade": "*",
+    "jquery": "^2.2.3",
+    "jwt-simple": "^0.3.1",
+    "method-override": "^2.3.5",
+    "moment": "^2.13.0",
     "mongodb": ">=0.9.6-7",
     "mongoose": ">=3.6",
-    "bcrypt": "^0.8.5",
```
Comment (wimoMisterX) [inline root/package.json]: Nothings changed it's just things moved up and down

File: root/server.js [hunk 2]
```diff
@@ -18,24 +18,22 @@ var jwt             = require('jwt-simple');
...
 app.set('port', process.env.PORT || 3000);
 app.set('views', __dirname + '/views');
 app.set('view engine', 'jade');
 
+app.use(passport.initialize());
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
+app.use(morgan('dev'));
 app.use(express.favicon());
-app.use(express.logger('dev'));
```
Comment (wimoMisterX) [inline root/server.js]: These already initialised in app.use(morgan('dev')); (mogran is the new logger (name has changed))

File: root/server.js [hunk 3]
```diff
@@ -18,24 +18,22 @@ var jwt             = require('jwt-simple');
...
-app.use(express.bodyParser());
```
Comment (wimoMisterX) [inline root/server.js]: You have already initalised this in 
 +app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
same thing

File: root/routes/index.js [hunk 4]
```diff
@@ -1,6 +1,3 @@
-const fs = require('fs');
-const path = require('path');
-
```
Comment (Prodge) [inline root/routes/index.js]: Good stuff cleaning these imports up

File: root/server.js (hunk 3 above)
Comment (Prodge) [inline root/server.js]: Nice :)
I have been meaning to go through and clean this stuff up!

**Comment by Prodge** [comment]
Really awesome work :)

**Pull request merged**